include/deal.II/grid/grid_generator.h

```diff
@@ -749,10 +749,16 @@ namespace GridGenerator
    * <table align="center" class="doxtable">
    *   <tr>
    *     <td>
-   *       <img src="hyper_ball_balanced_2d.png" alt="" width="40%">
+   *       \htmlonly <style>div.image
+   *         img[src="hyper_ball_balanced_2d.png"]{width:40%}</style>
+   *       \endhtmlonly
+   *       @image html hyper_ball_balanced_2d.png
```

Comment:
I assume this is because an ``<img>`` tag won't make doxygen export the .png and make it available online.